client/src/components/roulette/RouletteWheel.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Wheel } from 'react-custom-roulette';
+import { RouletteWheelContainer, SpinButton } from './roulette.styled.js';
+
+
+const data = [
+  { option: '0', style: { backgroundColor: 'green' } },
+  { option: '1' },
+  { option: '2' },
+  { option: '3' },
+  { option: '4' },
+  { option: '5' },
+  { option: '6' },
+  { option: '7' },
+  { option: '8' },
+  { option: '9' },
+  { option: '10' },
+  { option: '11' },
+  { option: '12' },
+  { option: '13' },
+  { option: '14' },
+  { option: '15' },
+  { option: '16' },
+  { option: '17' },
+  { option: '18' },
+  { option: '19' },
+  { option: '20' },
+  { option: '21' },
+  { option: '22' },
+  { option: '23' },
+  { option: '24' },
+  { option: '25' },
+  { option: '26' },
+  { option: '27' },
+  { option: '28' },
+  { option: '29' },
+  { option: '30' },
+  { option: '31' },
+  { option: '32' },
+  { option: '33' },
+  { option: '34' },
+  { option: '35' },
+  { option: '36' }
+]
+
+const backgroundColors = ['#9c0505', '#000000'];
+const textColors = ['white'];
+const outerBorderColor = 'black';
+const outerBorderWidth = 18;
+const innerBorderColor = '#000000';
+const innerBorderWidth = 29;
+const innerRadius = 42;
+const radiusLineColor = '#e3c13b';
+const radiusLineWidth = 4;
+const fontSize = 17;
+const textDistance = 77;
+const spinDuration = 1.0;
+
+export default function RouletteWheel () {
+  const [mustSpin, setMustSpin] = useState(false);
+  const [prizeNumber, setPrizeNumber] = useState(0);
+  const [winningNumInfo, setWinningNumInfo] = useState([]);
+
+  const handleSpinClick = () => {
+    const newPrizeNumber = Math.floor(Math.random() * data.length);
+    setPrizeNumber(newPrizeNumber);
+    // need to send newPrizeNumber to the backend when clicked
+    console.log(newPrizeNumber)
+    setMustSpin(true);
+    // do axios calls to get winning number information
+    // grabWinningNum(newPriceNumber);
+
+    <Celebration/>
+  };
+
+  const grabWinningNum = (num) => {
+    // how does router.jsx plays a role here?
```

Comment:
any calls that start with /api get routed to server/routes.js, any other request is seen as the client looking for an html file / website route